### PR TITLE
port LDAP advanced plugin hub improvements to community plugin

### PR DESCRIPTION
**Pull request by tlblessing**
https://konghq.atlassian.net/browse/DOCS-1353

Per https://konghq.atlassian.net/browse/DOCS-1269 as part of Shane's epic ticket https://konghq.atlassian.net/browse/DOCS-1162

This content was already approved by Darren and Lena in the advanced ldap plugin.

Parity as much as possible between community and enterprise docs.

Direct review link:

https://deploy-preview-2380--kongdocs.netlify.app/hub/kong-inc/ldap-auth/

### CODE REVIEW

File: app/_hub/kong-inc/ldap-auth-advanced/index.md [hunk 1]
```diff
@@ -120,7 +120,7 @@ params:
       default: "`ldap`"
       value_in_examples: ldap
       description: |
-        An optional string to use as part of the Authorization header. By default, a valid Authorization header looks like this: `Authorization: ldap base64(username:password)`. If `header_type` is set to "basic" then the Authorization header would be `Authorization: basic base64(username:password)`. Note that `header_type` can take any string, not just `"ldap"` and `"basic"`.
+        An optional string to use as part of the Authorization header. By default, a valid Authorization header looks like this: `Authorization: ldap base64(username:password)`. If `header_type` is set to "basic", then the Authorization header would be `Authorization: basic base64(username:password)`. Note that `header_type` can take any string, not just `"ldap"` and `"basic"`.
```
Comment (tlblessing) [inline app/_hub/kong-inc/ldap-auth-advanced/index.md:123]: missed this in original PR

File: app/_hub/kong-inc/ldap-auth/index.md [hunk 2]
```diff
@@ -108,36 +111,50 @@ params:
...
 ## Usage
 
-In order to authenticate the user, client must set credentials in `Proxy-Authorization` or `Authorization` header in following format
+To authenticate a user, the client must set credentials in either the
+`Proxy-Authorization` or `Authorization` header in the following format:
 
     credentials := [ldap | LDAP] base64(username:password)
 
-The plugin will validate the user against the LDAP server and cache the credential for future requests for the duration specified in `config.cache_ttl`.
+The Authorization header would look something like:
```
Comment (tlblessing) [inline app/_hub/kong-inc/ldap-auth/index.md:142]: This update is the crux of the ticket

File: app/_hub/kong-inc/ldap-auth/index.md [hunk 3]
```diff
@@ -108,36 +111,50 @@ params:
...
+
+    Authorization:  ldap dGxibGVzc2luZzpLMG5nU3RyMG5n
```
Comment (tlblessing) [inline app/_hub/kong-inc/ldap-auth/index.md:144]: This update is the crux of the ticket

File: app/_hub/kong-inc/ldap-auth/index.md [hunk 4]
```diff
@@ -108,36 +111,50 @@ params:
...
+
+The plugin validates the user against the LDAP server and caches the
+credentials for future requests for the duration specified in
+`config.cache_ttl`.
+
+You can set the header type `ldap` to any string (such as `basic`) using
```
Comment (tlblessing) [inline app/_hub/kong-inc/ldap-auth/index.md:150]: This update is the crux of the ticket

File: app/_hub/kong-inc/ldap-auth/index.md [hunk 5]
```diff
@@ -3,9 +3,11 @@ name: LDAP Authentication
 publisher: Kong Inc.
 version: 2.2.0
 
-desc: Integrate Kong with a LDAP server
+desc: Integrate Kong with an LDAP server
 description: |
-  Add LDAP Bind Authentication to a Route with username and password protection. The plugin will check for valid credentials in the `Proxy-Authorization` and `Authorization` header (in this order).
+  Add LDAP Bind Authentication to a Route with username and password protection. The plugin
+  checks for valid credentials in the `Proxy-Authorization` and `Authorization` header
```
Comment (lena-larionova) [inline app/_hub/kong-inc/ldap-auth/index.md:9]:
```suggestion
  checks for valid credentials in the `Proxy-Authorization` and `Authorization` headers
```

**Review by lena-larionova — APPROVED** [review]
One minor edit, otherwise LGTM

**Comment by tlblessing** [comment]
> One minor edit, otherwise LGTM

Good catch, thanks!

**Pull request merged**